src/components/shapes/display_outlines.js

```diff
@@ -249,8 +249,8 @@ module.exports = function displayOutlines(polygons, outlines, dragOptions, nCall
                     var dx = nextPoint[1] - x;
                     var dy = nextPoint[2] - y;
 
-                    var width = dy ? 5 : Math.max(Math.min(25, Math.abs(dx) - 5), 5);
-                    var height = dx ? 5 : Math.max(Math.min(25, Math.abs(dy) - 5), 5);
+                    var width = dx ? 5 : Math.max(Math.min(25, Math.abs(dx) - 5), 5);
+                    var height = dy ? 5 : Math.max(Math.min(25, Math.abs(dy) - 5), 5);
```

Comment:
Thanks for the PR. It looks like a good catch.
But still this logic here doesn't make sense to me. 
If there is no `dx` or `dy` why we take the absolute of those?

Reply:
I think you are right... I didn't even notice that that is backwards too. I think it should be:

```javascript
var width = dy ? Math.max(Math.min(25, Math.abs(dx) - 5), 5): 5;
var height = dx ? Math.max(Math.min(25, Math.abs(dy) - 5), 5): 5;
```

Does that make sense?

Reply:
I'll update the PR.

Reply:
That is a tough request, probably beyond my skills (I'm really not a javascript developer). The issue appears when using react-plotly. But maybe I can trigger it with a local build of plotly. If I can figure out how to use a local build...

Reply:
FYI - The plotly.js bundles would be available in the `ARTIFACTS` tab of each push on `CircleCI` in few minutes.
For now you could see the progress at https://app.circleci.com/pipelines/github/plotly/plotly.js/8553/workflows/5d643013-8f41-4074-ad75-d3cf8ea9aa99/jobs/189119/steps

Reply:
OK here they are: 

A minified bundle: https://output.circle-artifacts.com/output/job/a43ea437-79a0-4950-8e2f-1cb2606980aa/artifacts/0/dist/plotly.min.js

An unminified bundle: https://output.circle-artifacts.com/output/job/a43ea437-79a0-4950-8e2f-1cb2606980aa/artifacts/0/dist/plotly.js

Reply:
@dsblank Testing your branch locally, I am afraid you are not editing the correct place.
The controllers added in `src/components/shapes/display_outlines.js` are for `editable` shapes.
But it looks like in the bug report you are referring to drag elements at the end of each axis.
Is that correct?

Reply:
Not just the drag elements at the end, but more of the <rect> elements. If I manually edit the <rect>'s width, height, x, and y, then it works (eg, I can click on a scatter point). Otherwise, I am unable to click because some cover the plot.

Reply:
You are probably correct that I am not fixing the correct thing. Perhaps there is a similar bug in another section that defines the rects.

Reply:
I did verify that the code in this PR now matches the rest of plotly:

```bash
$ git grep "dx ?"
dist/plotly-basic.js:          var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5) : 5;
dist/plotly-cartesian.js:          var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5) : 5;
dist/plotly-finance.js:          var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5) : 5;
dist/plotly-geo.js:          var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5) : 5;
dist/plotly-gl2d.js:          var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5) : 5;
dist/plotly-gl3d.js:          var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5) : 5;
dist/plotly-mapbox.js:          var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5) : 5;
dist/plotly-strict.js:          var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5) : 5;
dist/plotly-with-meta.js:          var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5) : 5;
dist/plotly.js:          var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5) : 5;
src/components/shapes/display_outlines.js:                    var width = dx ? Math.max(Math.min(25, Math.abs(dx) - 5), 5): 5;
```